lib/ansible/modules/monitoring/zabbix/zabbix_action.py

```diff
@@ -1107,6 +1112,8 @@ def construct_the_data(self, operations):
             list: constructed operation data
         """
         constructed_data = []
+        if operations is None:
+            return None
```

Comment:
As we already have `default=[]` for operations, there is no need for this block anymore.